config_defaults/subtests/docker_cli/negativeusage.ini

```diff
@@ -70,24 +70,6 @@ subarg = -e,PATH=/tmp,%%(FQIN)s,true
 stderr = exec\: \"true\"\: executable file not found in \$PATH
 extcmd = 127
 
-[docker_cli/negativeusage/iv5]
-subcmd = run
-subarg = --tmpfs,/dev,%%(FQIN)s
-stderr =  no such file or directory
-extcmd = 127
-
-[docker_cli/negativeusage/iv6]
-subcmd = run
-subarg = --tmpfs,:rw,%%(FQIN)s
-stderr =  open /proc/self/task/1/attr/exec: no such file or directory
-extcmd = 125
-
-[docker_cli/negativeusage/iv7]
-subcmd = run
-subarg = --tmpfs,.,%%(FQIN)s
-stderr =  Message: Failed to open /dev/null - open /dev/null: permission denied
-extcmd = 125
```

Comment:
@edsantiago 

Okay, with two bugs open now on ``iv7``, and #655 merged, let's keep the iv7 test - though it will need to be updated once the bug(s) are fixed.  The others (``iv5`` and ``iv6``) can go.  To be considered for adding back in when/if https://trello.com/c/tdAhquxo is completed.  Sound reasonable?